R/check_description.R

```diff
@@ -141,7 +167,25 @@ unchanged_repo <- function(repo, old_status) {
     )
 }
 
-#' Check the license
+#' Check the license of a package
+#'
+#' Every package need a clear license.
+#' Without a license, the end-users have no clue under what conditions they can
+#' use the package.
+#' You must specify the license in the `DESCRIPTION` and provide a `LICENSE.md`
+#' file.
+#'
+#' @details
+#' This functions checks if the `DESCRIPTION` mentions one of the standard
+#' licenses.
+#' The `LICENSE.md` must match this license.
+#' Use `setup_package()` to add the correct `LICENSE.md` to the package.
```

Comment:
Neither in the documentation of `setup_package()`, nor in the vignette referred to, any information is given on how to choose for the desired license (or actually there is no information on licenses at all).  For now it is obviously not a problem, but make sure this is covered as soon as it is needed.

Reply:
Currently checklist picks a license for you. GPL-3 for packages and CC-BY-SA 4.0 for source repos. We should have a discussion about this at INBO.

Reply:
My apologies, in the documentation of `setup_package()` (the part in the template), it is explicitly mentioned that a GPL-3 license is added.  To be changed after the discussion, for now the current status seems well documented to me.